src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,18 @@
+package wolox.training.repositories;
+
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import wolox.training.models.Book;
+
+@Repository
+public interface BookRepository extends JpaRepository<Book, Long> {
+
+    /**
+     * Method that obtain a book by its author
+     * @param author: Name of author (String)
+     * @return {@link Book}
+     */
+    Optional<Book> findFirstByAuthor(String author);
```

Comment:
I think you can use directly findByAuthor, what do you think?

Reply:
It is correct, i do not know why I came to think that when consulting by author it would give me a list, that's why I added findFirst.

Reply:
Solved.